config.xml

```diff
@@ -155,13 +154,11 @@
     </universal-links>
     <plugin name="com.dbaq.cordova.contactsPhoneNumbers" spec="~0.0.10" />
     <engine name="android" spec="~7.0.0" />
-=======
     <plugin name="cordova-plugin-media-capture" spec="^1.4.3">
         <variable name="CAMERA_USAGE_DESCRIPTION" value="let me use camera plz!" />
         <variable name="MICROPHONE_USAGE_DESCRIPTION" value="let me use mic plz!" />
     </plugin>
     <plugin name="cordova-plugin-console" spec="^1.1.0" />
     <engine name="android" spec="^7.0.0" />
```

Comment:
you have <engine name="android"  twice.

Please sort this file in some way, e.g., all <plugin> at the end, before <engine> , before <preference>, before <platform ...




<engine name="android" spec="~7.0.0" />
    <plugin name="cordova-plugin-media-capture" spec="^1.4.3">
        <variable name="CAMERA_USAGE_DESCRIPTION" value="let me use camera plz!" />
        <variable name="MICROPHONE_USAGE_DESCRIPTION" value="let me use mic plz!" />
    </plugin>
    <plugin name="cordova-plugin-console" spec="^1.1.0" />
    <engine name="android" spec="^7.0.0" />